onboarding/02_development_environment.md

```diff
@@ -64,6 +64,24 @@ To download Postgres: http://oscg-downloads.s3.amazonaws.com/packages/PostgreSQL
 
 For further instructions: https://medium.com/@colinrubbert/installing-ruby-on-rails-in-windows-10-w-bash-postgresql-e48e55954fbf
 
+## Tech Stack
+
+Here are the technologies used in the project, along with some tutorials if you'd like to learn.
+
+| Tech | Version | Purpose | Getting Started |
+|------|---------|---------|-----------------|
+| Git | 2.4+ | Version control | [Udacity course](https://classroom.udacity.com/courses/ud775), [good comprehensive online book](https://git-scm.com/book/en/v2) |
+| Postgres | 9.6 | Database | [Tutorial](https://www.postgresql.org/docs/8.0/static/tutorial.html) |
+| SQL | |  Language used for database queries | [Tutorial from Postgres site](https://www.postgresql.org/docs/8.0/static/tutorial-sql.html) |
+| Python | v3 | Data Analysis & Webserver | [Tutorial](https://docs.python.org/3/tutorial/) |
```

Comment:
Do you mind linking to Anaconda's distribution of Python? That's what we've been using so far and I still feel like that's the easiest way to get everyone on the same dev environment.

Reply:
I've found this git reference to be really helpful as I got started: http://rogerdudler.github.io/git-guide/